### ISSUE TITLE
Group video call is not visible

### ISSUE THREAD

**Issue by mhlavac** [RocketChat/Rocket.Chat#3413]
If I start a group video call, I don't see the video and controls.

I've checked the HTML and if I change webrtc css to:

```
position: relative;
z-index: 20;
```

then I see videos and controls.

I'll try to open pull request for this tonight, but if someone can fix this and knows how to fix this properly then it would be great to do it sooner.

**Comment by xadhoom** [comment]
mmmh I have the same issue, but to properly display it I have to remove position:absolute from .flex-tab .content>.animated

**Comment by xadhoom** [comment]
I've added
```
.flex-tab .content>.animated {
  position: initial;
}
```
under custom css into admin section and now it displays correctly

**Comment by mhlavac** [comment]
@xadhoom great idea. I didn't realize I can hotfix it in layout css like this. Thanks

**Comment by xadhoom** [comment]
yes, but this hack breaks the sidenav user details (the big panel that appears when you click on a user)

**Comment by xadhoom** [comment]
well, in this way:

```
.flex-tab .content>.animated {
  position: initial;
}

.user-view.animated {
  position: absolute !important;
}
```

it works, but , hey, is really a dirty hack :)

**Comment by Sing-Li** [comment]
New solution testing this week.

On https://demo.rocket.chat

New solution testing now.

On https://demo.rocket.chat

When you are in a Direct Message or Private Group. You will notice a new video button on the toolbar , just click it anytime to start a video conference with all those in the room

**Comment by joshua5822** [comment]
Has there been any progress on this issue?

**Comment by TheSovietCitizen** [comment]
Hello, Rocket team!
I have the same problem. When I start a group video call  "I don't see the video and controls." ©
What can I do to fix it?

**Comment by qk4l** [comment]
Seems it is not resolved in 0.36.
I had to add custom CSS.

**Comment by madumlao** [comment]
I created a user style that incorporates the workaround above. Still needed in current rocketchat beta hosting.
https://userstyles.org/styles/134454/rocketchat-group-call-fix

**Comment by rasos** [comment]
Group Video with WebRTC still not working in 0.49.2.
1-1 works, also Jitsi works, but the latter is difficult to install.

**Comment by flyfishy** [comment]
Group Video with WebRTC not working in 0.50.1. Same as above, Direct and Jitsu work great.
While in Private Channel, Member list (sidebar icon) is clicked, Member List opens, and at the top of the list shows the Video button. When clicked, "Group video p-group-test do want to accept?" clicking yes turns on camera's, and audio can be heard, but no video or controls is shown. And the only way to stop, is to exit Browser, or Reload app.

**Comment by travka** [comment]
Still not working in macos app

**Comment by KSchreyer** [comment]
Just updated to version 0.54.2. Behaviour is still as @flyfishy described it.

**Comment by KSchreyer** [comment]
I think I've found the cause for the invisible WebRTC div container.

On a 1-on-1 WebRTC video call, the class **`animated-hidden`** is added to the DOM-Element `<div class="list-view animated">`. This must be also done for a private group WebRTC video call and the video-container gets visible.

So the CSS class must be added at some point where the private group WebRTC call gets invoked and later on removed when the WebRTC call shuts down.

After a qick search in the code repository, it seems that the method `SideNav.openFlex()` is used to add this css class.

**Comment by KSchreyer** [comment]
Does anybody know where to fix this in the code?
I'm currently do not know enough of the Meteor/Rocket.Chat internas, but when someone pointing me to the right spot, I'm maybe able to fix this and submit a pull request.

@geekgonecrazy: Can you change this from _type: improvement_ to _type: bug_, because that is what it is. You click on group chat, your camera led turns on and sound is going over the wire, but unfortunately you see nothing.

**Comment by dasjosch** [comment]
Has there been any progress on this issue?

**Comment by TheAngryFox** [comment]
Having exactly the same issue! Has there been any progress with this issue recently?

**Comment by salvino72** [comment]
I just inserted this CSS edit from the administration panel.
/Admin /Layout in Custom CSS

```
.flex-tab-container .flex-tab .content>.animated {
    position: initial!important;
  }
```

**Comment by 12test12** [comment]
@salvino72 thank you your solution works for me.

**Comment by dkoryavov** [comment]
Is there any chance that the bug will be fixed soon?
